jetstream/desktop-credit-card-autofill-global-enablement-holdback.toml

```diff
@@ -19,7 +19,7 @@ from_expression = """(
     `mozdata.firefox_desktop.events_stream`
   WHERE
     event_category IN ('creditcard')
-    AND event_name IN ('detected_cc_form_v2', 'filled_cc_form_v2', 'submitted_cc_form_v2')
+    AND event_name IN ('detected_cc_form_v2', 'filled_cc_form_v2', 'submitted_cc_form_v2', 'popup_shown_cc_form_v2)
```

Comment:
Typo here: `'popup_shown_cc_form_v2` missing the closing `'`

Reply:
Also, is there any reason to have all these `event_name` values? It seems like you're only using the one (`popup_shown_cc_form_v2`).